lib/remote/apilistener.cpp

```diff
@@ -432,9 +432,14 @@ void ApiListener::ListenerCoroutineProc(boost::asio::yield_context yc, const Sha
 
 	for (;;) {
 		try {
-			auto sslConn (Shared<AsioTlsStream>::Make(io, *sslContext));
+			asio::ip::tcp::socket socket(io);
+
+			server->async_accept(socket.lowest_layer(), yc);
 
-			server->async_accept(sslConn->lowest_layer(), yc);
+			UpdateSSLContext();
```

Comment:
You update it on every new connection independent of whether the CRL has changed or not. Are you sure this is needed?